### PR TITLE
11-9-22 core release notes

### PR DESCRIPTION
**Pull request by jseldess**
- Add docs for weekly release notes
- Update SQL support doc for changes in release

### CODE REVIEW

File: docs/releases/readyset-core.md [hunk 1]
```diff
@@ -2,12 +2,87 @@
...
+        ```
+
+    - ReadySet Adapter (linux-x84_64)
+        ``` sh
+        docker pull public.ecr.aws/readyset/readyset-adapter:beta-2022-11-09
+        ```
+
+### Changes
+
+- Added replication and caching support for Postgres enumerated data types. [d8e428d](https://github.com/readysettech/readyset/commit/d8e428d5cf63fc704d2b9242066c2bccafffd76b), [e282ae9](https://github.com/readysettech/readyset/commit/e282ae99fe0f7e2491f56203b455b9f2e72ebd8e), [40d217f](https://github.com/readysettech/readyset/commit/40d217fdb8b55f004a8e3cbca9d2e30b379bd269), [ef95276](https://github.com/readysettech/readyset/commit/ef95276d4c3498ce77eed81e2d2d2185c0b21791) <!-- Griffin -->
```
Comment (glittershark) [inline docs/releases/readyset-core.md]: we also support `ALTER TYPE ... RENAME TO` (https://github.com/readysettech/readyset/commit/23e6a120ac3090cdd9cdf408824b24973ebc80d6), `ALTER TYPE ... RENAME VALUE ...` (https://github.com/readysettech/readyset/commit/312dfa04574cd7d471d5b35598c92df13d496168), and `ALTER TYPE... SET SCHEMA`

File: docs/releases/readyset-core.md [hunk 2]
```diff
@@ -2,12 +2,87 @@
...
+- Fixed a bug that would cause `CREATE CACHE` commands to fail for MySQL `JOIN` queries with join conditions in the `WHERE` clause instead of the `ON` clause. [f9be6d6](https://github.com/readysettech/readyset/commit/f9be6d6e99c284f2a30163075a272063bab2f449) <!-- Nick -->
+- Upgraded OpenSSL to address a recently identified [vulnerability](https://github.com/advisories/GHSA-8rwr-x37p-mx23). [4525073](https://github.com/readysettech/readyset/commit/4525073eb2a505ef0ef2489f79a0149cf6a29fcd) <!-- Griffin -->
+- ReadySet now logs the name of table/views that fail to snapshot or replicate. [7ffaf07](https://github.com/readysettech/readyset/commit/7ffaf07cbfca81691760b3ebf0b53afc66ac85e0) <!-- Luke -->
+- Added node column types to Graphviz data flow visualizations. [30f023b](https://github.com/readysettech/readyset/commit/30f023bf41c9c6a0548b83f71e76c35ee7f336a0) <!-- Griffin -->
+- When starting a ReadySet Adapter, if its version does not match the ReadySet Server, the Adapter now fails to start and returns and error. [e00c742](https://github.com/readysettech/readyset/commit/e00c742bdee9dd7c23361cade496fee1402ecaa3) <!-- Luke -->
```
Comment (glittershark) [inline docs/releases/readyset-core.md]:
```suggestion
- When starting a ReadySet Adapter, if its version does not match the ReadySet Server, the Adapter now fails to start and returns an error. [e00c742](https://github.com/readysettech/readyset/commit/e00c742bdee9dd7c23361cade496fee1402ecaa3) <!-- Luke -->
```

File: docs/releases/readyset-core.md (hunk 1 above)
Comment (jseldess) [reply docs/releases/readyset-core.md]: Done.

File: docs/releases/readyset-core.md (hunk 2 above)
Comment (jseldess) [reply docs/releases/readyset-core.md]: Done.

File: docs/releases/readyset-core.md [hunk 3]
```diff
@@ -2,12 +2,81 @@
...
+- Added caching support for the `?|` and `?&` JSON operators. [f5ae4d](https://github.com/readysettech/readyset/commit/f5ae4d0c004dfe9095523155cf0ceb78fe14309b), [6598431](https://github.com/readysettech/readyset/commit/659843199836c321ed4fecbce31c894dd2100fa2) <!-- Nick -->
+- Fixed a bug that would cause `CREATE CACHE` commands to fail for MySQL `JOIN` queries with join conditions in the `WHERE` clause instead of the `ON` clause. [f9be6d6](https://github.com/readysettech/readyset/commit/f9be6d6e99c284f2a30163075a272063bab2f449) <!-- Nick -->
+- Upgraded OpenSSL to address a recently identified [vulnerability](https://github.com/advisories/GHSA-8rwr-x37p-mx23). [4525073](https://github.com/readysettech/readyset/commit/4525073eb2a505ef0ef2489f79a0149cf6a29fcd)
+- ReadySet now logs the name of table/views that fail to snapshot or replicate. [7ffaf07](https://github.com/readysettech/readyset/commit/7ffaf07cbfca81691760b3ebf0b53afc66ac85e0) <!-- Luke -->
```
Comment (lukoktonos) [inline docs/releases/readyset-core.md]:
```suggestion
- ReadySet now logs the names of tables/views that fail to snapshot or replicate. [7ffaf07](https://github.com/readysettech/readyset/commit/7ffaf07cbfca81691760b3ebf0b53afc66ac85e0) <!-- Luke -->
```

Comment (jseldess) [reply docs/releases/readyset-core.md]: Done.

**Review by imeyer — APPROVED** [review]
LGTM

**Pull request merged**